#!/usr/bin/env python
PACKAGE = "robosys_ros"

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("hue_upper", int_t, 0, "", 5, 0, 180)
gen.add("hue_lower", int_t, 0, "", 175, 0, 180)
gen.add("saturation_upper", int_t, 0, "", 255, 0, 255)
gen.add("saturation_lower", int_t, 0, "", 150, 0, 255)
gen.add("value_upper", int_t, 0, "", 255, 0, 255)
gen.add("value_lower", int_t, 0, "", 135, 0, 255)

exit(gen.generate(PACKAGE, "robosys_ros", "Red"))
